examples/source/interactivity-dynamic-content/Article_Bookmark_Email/api.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const express = require('express');
+const multer = require('multer');
+const upload = multer();
+
+// eslint-disable-next-line new-cap
+const examples = express.Router();
+
+examples.post('/submit-form-bookmark', upload.none(), (request, response) => {
```

Comment:
What's the purpose of multer here?

Reply:
The id of the bookmarked article is passed along, but multer is probably a bit overpowered to handle that request. Thanks for the input.